superset/assets/src/visualizations/table.css

```diff
@@ -1,5 +1,5 @@
 .slice-grid .widget.table .slice_container {
```

Comment:
possibly good news: anything with `.widget` I think can be removed. this is what grid cells on the old dashboard used to be called.